contracts/core/Modifier.sol

```diff
@@ -6,7 +6,7 @@ pragma solidity >=0.7.0 <0.9.0;
 import "./IExecutor.sol";
 import "./Module.sol";
 
-contract Modifier is Module {
+abstract contract Modifier is Module {
```

Comment:
This has been changed because in the `Modifier` we are not implementing the `setUp` function